Resources/Prototypes/Catalog/VendingMachines/cola.yml

```diff
@@ -6,11 +6,10 @@
   spriteName: cola
   startingInventory:
     DrinkColaCan: 10
+    DrinkGrapeCan: 10
     DrinkIceTeaCan: 10
     DrinkLemonLimeCan: 10
     DrinkGrapeCan: 10
     DrinkSpaceMountainWindCan: 10
     DrinkSpaceUpCan: 10
     DrinkStarkistCan: 10
-    DrinkThirteenLokoCan: 10
```

Comment:
![grafik](https://user-images.githubusercontent.com/16618384/108610628-769b8a80-73d7-11eb-904b-0d666e22c0b2.png)

Reply:
I am ashamed. No idea how that happened from copying to excel, hitting sort and copying back.

Resolved anyhow.